docs/basic-docs/atomicdex-api-20-dev/enable_z_coin_init.md

```diff
@@ -0,0 +1,77 @@
+
+## task::enable_z_coin::init
+
+Z coins, like Pirate (ARRR) and the test coin ZOMBIE take a little longer to enable, and use a new two step method to enable. Activation can take a little while the first time, as we need to download some block cache data, and build a wallet database. Subsequent enabling will be faster, but still take a bit longer than other coins. The second step for activation is optional, but allows us to check the status of the activation process.
+To enable Z coins you also need to [install some Zcash Params](https://forum.komodoplatform.com/t/installing-zcash-params/603)
+
+
+#### Arguments
+
+| Structure                                 | Type            | Description                                                                                                                                                          |
+| ----------------------------------------- | --------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ticker                                    | string          | Ticker of coin to activate                                                                                                                                           |
+| activation_params                         | object          | List of [Electrum servers](https://github.com/KomodoPlatform/coins/tree/master/electrums)                                                                            |
+| activation_params.mode.zcash_params_path  | string          | Optional. Path to folder containing [Zcash parameters](https://z.cash/technology/paramgen/). Defaults to standard location as defined in [this guide](https://forum.komodoplatform.com/t/installing-zcash-params/603) |
```

Comment:
We should provide a description of `zcash_params_path` after `activation_params.mode`, as its optional parameter and its not a part of `mode`. `zcash_params_path` is one of `activation_params` arguments. 
You can expand the view till activation_params [here](https://codebeautify.org/string-to-json-online/y22489f99)
In addition, you can see, how zcoin activation params look in API code
```rust
pub struct ZcoinActivationParams {
    pub mode: ZcoinRpcMode,
    pub required_confirmations: Option<u64>,
    pub requires_notarization: Option<bool>,
    pub zcash_params_path: Option<String>,
}
```
Where `ZcoinRpcMode`could be `Native` or `Light` with `electrum_servers` and with `light_wallet_d_servers`.